test/compliance/SELECT.test.js

```diff
@@ -159,6 +159,34 @@ describe('SELECT', () => {
       })
     })
 
+    test('expand to many with 200 columns', async () => {
+      const nulls = length => new Array(length).fill().map((_, i) => ({ as: `null${i}`, val: null }))
+      const cqn = {
+        SELECT: {
+          from: { ref: ['complex.Authors'] },
+          columns: [{ ref: ['ID']}, { ref: ['name']}, { ref: ['books'], expand: ['*', ...nulls(197)]}]
+        },
+      }
+
+      const res = await cds.run(cqn)
+      // ensure that all null values are returned and as json format
+      assert.strictEqual(Object.keys(res[0].books[0]).length, 200)
```

Comment:
from the comment it should be checked whether the `nulls` is returned in json format. Is the check `.length` enough to make sure that this holds true?

Reply:
I'll adapt the comment. only the length is important.